daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md

```diff
@@ -0,0 +1,115 @@
+```
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+## Spec metadata fields
+| Field              | Required | Binding support | Details | Example |
+|--------------------|:--------:|--------|--------|---------|
+| endpoint                | Y        | Input/Output |the endpoint to get Nacos server addresses | `"******.mse.aliyuncs.com:8848"`
+| region                | Y        | Input/Output |the regionId for kms | `"cn-shanghai"`
+| namespace                | Y        | Input/Output |the namespaceId of Nacos | `"e525eafa-f7d7-4029-83d9-************"`
+| config                | Y        | Output |A comma separated string of key | `"key1:group1"`
+| accessKey                | N        | Input/Output |the AccessKey for kms| `"****************"`
+| secretKey                | N        | Input/Output |the SecretKey for kms | `"****************"`
+| timeout                | N        | Input/Output |timeout for requesting Nacos server, default value is 10000 ms | `1000`
+| cacheDir                | N        | Input/Output |the directory for persist Nacos service info,default value is current path | `"/tmp/nacos/cache"`
+| updateThreadNum                | N        | Input/Output |the number of gorutine for update Nacos service info,default value is 20 | `20`
+| notLoadCacheAtStart                | N       | Input/Output |not to load persistent Nacos service info in CacheDir at start time | `"false"`
+| updateCacheWhenEmpty                | N        | Input/Output |update cache when get empty service instance from server | `"false"`
+| username                | N        | Input/Output |the password for Nacos auth | `"username"`
+| password                | N        | Input/Output |the password for Nacos auth | `"password"`
+| logDir                | N        | Input/Output |the directory for log, default is current path | `"/tmp/nacos/log"`
+| rotateTime                | N        | Input/Output |the rotate time for log, eg: 30m, 1h, 24h, default is 24h| `"1h"`
+| maxAge                | N        | Input/Output |the max age of a log file, default value is 3 | `3`
+| logLevel                | N        | Input/Output |the level of log, it's must be debug,info,warn,error, default value is info | `"info"`
+| watches                | N        | Input |A comma separated string of watch keys | `"key1:group1,key2:group2"`
+
+## Binding support
+
+This component supports both **input and output** binding interfaces.
+
+This component supports **output binding** with the following operations:
+- `create`
+- `get`
+
+## Create an AliCloud Nacos
+Follow the instructions [here](https://www.alibabacloud.com/help/doc-detail/59963.htm) on setting up AliCloud nacos.
+Follow the instructions [here](https://help.aliyun.com/document_detail/85466.html) on setting up AliCloud nacos.
```

Comment:
```suggestion
Follow [these instructions](https://www.alibabacloud.com/help/doc-detail/59963.htm) to setup AliCloud Nacos.
   - A Chinese (simplified) version can be found [here](https://help.aliyun.com/document_detail/85466.html)
```

Added suggestion for grammer. But curious why these links go to ACM. Shouldn't the Nacos link be https://nacos.io/en-us/docs/quick-start.html?